httpie/config.py

```diff
@@ -128,3 +128,7 @@ def __init__(self, directory: Union[str, Path] = DEFAULT_CONFIG_DIR):
     @property
     def default_options(self) -> list:
         return self['default_options']
+
+    @property
+    def plugins_dir(self) -> Path:
+        return Path(self.get('plugins_dir', self.directory / 'plugins')).resolve()
```

Comment:
I don't think `~/.config/httpie/plugins` is the right direcctory to install the plugins into, they aren't really "configs", would better fit in `~/.local/share` or `~/.cache`

Discussion: https://discord.com/channels/725351238698270761/801485021016752208/914132385669472266

Reply:
This was discussed on discord.